pulsar-io/cassandra/src/main/java/org/apache/pulsar/io/cassandra/CassandraAbstractSink.java

```diff
@@ -98,7 +98,7 @@ private void createClient(String roots) {
                 b.withPort(Integer.parseInt(hostPort[1]));
             }
         }
-        cluster = b.build();
+        cluster = b.withoutJMXReporting().build();
```

Comment:
This is production code, but as mentioned in https://docs.datastax.com/en/developer/java-driver/3.11/manual/metrics/#metrics-4-compatibility and investigated our dependencies, I think it should never work...

If we meet real-world use case, we can add an option for configuring this behavior later.